app/src/main/java/mod/hey/studios/build/BuildSettings.java

```diff
@@ -14,7 +14,7 @@ public class BuildSettings extends ProjectSettings {
     public static final String SETTING_ENABLE_LOGCAT = "enable_logcat";
 
     public static final String SETTING_DEXER_D8 = "D8";
-    public static final String SETTING_DEXER_DX = "Dx";
+    public static final String SETTING_DEXER_DX = "DX";
```

Comment:
what

Reply:
the dexer is name is just "dx" so if you're having first letter capital, its better to have the whole capitalised

Reply:
if you're that picky about it, why change it in just one place? consistency is key.

anyway randomly changing preference values also breaks backwards-compatibility with existing configurations. this one is just a mild one since nothing explicitly checks if a preference equals `SETTING_DEXER_DX`, but e.g. other apps might depend on exactly this value.

so I'd say please just drop this change